plugin-clock/lxqtclock.cpp

```diff
@@ -162,14 +163,15 @@ void LxQtClock::showTime(const QDateTime &now)
     mRotatedWidget->update();
 }
 
-void LxQtClock::restartTimer(const QDateTime &now)
+void LxQtClock::restartTimer()
```

Comment:
On beginning or after config change... the content is updated and timer is restarted to fire right after time changes (each second or each minute).